.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         steps:
             - uses: actions/checkout@v3
             - uses: ./.github/actions/prepare-playground
-            - run: npx nx affected --target=test --configuration=ci
+            - run: node --expose-gc node_modules/nx/bin/nx affected --target=test --configuration=ci
```

Comment:
If this is needed for the memory leak test to pass then local tests should run with this flag, too. Otherwise that one test will always fail. Perhaps updating the `test` script in `package.json` would be a good start.

It would be also nice to assert that `--expose-gc` was used and provide a useful error message explaining how you need to run `node --expose-gc  node_modules/nx/bin/nx` and not just `nx`.

Reply:
@adamziel This is what I wanted to bring up in the meeting. Is there a way in `nx` to include this switch by default?

Reply:
@seanmorris not that I'm aware of. The [vitest executor](https://nx.dev/nx-api/vite/executors/test) doesn't list any node-specific options, neither does the [vitest](https://vitest.dev/config/) library itself. Running `nx` seems to be forking into a `nx` daemon process and an executor process where all node `execArgv` options are passed to the executor process. Other than that, I don't see a way to specify `execArgv` for the executor process in the general case. I guess that's something `nx` itself would need to support.

One think we could do is define the `test` command using the `nx:run-commands` executor, like this:

```json
"publish": {
			"executor": "nx:run-commands",
			"options": {
				"command": "node tools/scripts/publish.mjs php-wasm-node {args.ver} {args.tag}",
				"parallel": false
			},
			"dependsOn": ["build"]
		}
```

It could enforce `--expose-gc` but would just require some fiddling to find the right `vitest` command to run.

Reply:
@adamziel Hmmm, perhaps we should keep things working the way we've got it right now then, and revisit if we end up having issues with it.

I don't think that command is going to be called manually much.

Reply:
https://github.com/WordPress/wordpress-playground/pull/639/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R38